spec/tests/artifact.fmf

```diff
@@ -0,0 +1,29 @@
+summary: Files or directories to be collected after test execution
+
+story:
+    As a tester I want to collect files or directories produced by
+    test so I can process them later. Some of the collected artifacts
+    can be understood by tmt.
+
+description: |
+    Test can produced artifacts as coverage data, junit summaries,
+    coredumps or other interesting data. By listing them in this attribute
+    execute steps collects them from the environemnt where test is running and saves
+    it to the run directory.
```

Comment:
So this part is now already happenning: The test is executed in it's directory, as defined by `path`, and after test execution is completed `workdir` is pulled back from the guest using `rsync`. Do you propose some additinal processing?

Reply:
What if I store files outside of workdir? 
Note that current test aren't supposed to create any content in their workdir.

Maybe we need rethink and, at least for beakerlib tests support `rlFileSubmit`

Reply:
Or let executor to "submit" files - beakerlib (and restraint) allow test tu submit file. 
I'd like to have those file in 'artifacts' field.

So report plugin knows that something was submitted from the test.
The additional `dictionary` is to provide meaning to some files - Now I have xunit in mind, but let's make it dynamic.

So say there is `junit: file` in artifacts and I use `report --how junit` - plugin knows what files it should check for (parsing any file or arbitratry xml files doesn't seem good idea) and will process them.